charts/newrelic-logging/values.yaml

```diff
@@ -43,6 +44,39 @@ fluentBit:
   #     fieldRef:
   #       fieldPath: metadata.name
 
+  # Indicates how fluent-bit database is persisted
+  persistence:
+    # Define the persistent mode for fluent-bit db, allowed options are `hostPath` (default), `none`, `persistentVolume`.
+    # - `hostPath` will use hostPath to store the db file on the node disk.
+    # - `none` will disable the fluent-bit db file, this could cause log duplication or data loss in case fluent-bit gets restarted.
+    # - `persistentVolume` will use a ReadWriteMany persistent volume to store the db file. This will override `fluentBit.db` path and use `/db/${NODE_NAME}-fb.db` file instead.
```

Comment:
Ah, this is what I was looking for. Can you somehow display this in the readme as well?

Reply:
Will do

Reply:
Done! [2948836](https://github.com/newrelic/helm-charts/pull/1259/commits/2948836cfb7cf16788942de4139ff7e7ec02ef2e)